src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+   
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
move random to the class variables, like in example with magic number

Reply:
not fixed